frontend-maven-plugin/src/it/custom-install-directory/verify.groovy

```diff
@@ -1,2 +1,2 @@
-assert new File(basedir, 'target/node').exists() : "Node was not installed in the custom install directory";
+assert new File(basedir, 'target/.buildenv/node').exists() : "Node was not installed in the custom install directory";
```

Comment:
basically, it is your decision. However, the name of the temporary directory `node` is not a good choice especially, if you want to ignore it on commit. Therefore, I introduced a more speaking and generic folder name to also include further tmp local environments in future.
I even do not see any "backward-compatibility issues" here as this is just a folder of temporary files, which will even be recreated at build time, but correct me if I am wrong.

So I would highly recommend to change the folder name.